jib-core/src/main/java/com/google/cloud/tools/jib/builder/PullAndCacheBaseImageLayerStep.java

```diff
@@ -46,7 +47,7 @@
       BuildConfiguration buildConfiguration,
       Cache cache,
       DescriptorDigest layerDigest,
-      Future<Authorization> pullAuthorizationFuture) {
```

Comment:
Should probably change the private final field as well from `Future` to `ListenableFuture`

Reply:
Good catch - found a bunch of other places too